libnmstate/nm/connection.py

```diff
@@ -387,20 +387,25 @@ def _delete_connection_callback(self, src_object, result, user_data):
     @staticmethod
     def _commit_changes_callback(src_object, result, user_data):
         mainloop, nmdev = user_data
+        devname = src_object.get_interface_name()
         try:
             success = src_object.commit_changes_finish(result)
         except Exception as e:
             if mainloop.is_action_canceled(e):
                 logging.debug("Connection update aborted: error=%s", e)
             else:
+                if nmdev:
+                    devname = nmdev.props.interface
```

Comment:
Does this need to be assigned again here? Is the first assignment not enough?

Reply:
To be consistent, I think we need to take the state and name from the same source.
Taking the name from the device and the name from the profile seems wrong to me.

Reply:
In my opinion @EdDev is right. If we are taking the state from the device then we should also take the name from the device.